styles/customboot.less

```diff
@@ -673,3 +673,41 @@ img.grayscale {
     }
   }
 }
+
+.usa-icon {
+  width: 20px;
+  height: 20px;
+  margin: 5px;
+  background-image: url('/images/map/usa.svg');
+  background-size: 20px 20px;
+  background-repeat: no-repeat;
+  display: inline-block;
+}
+
+.multipleResults {
+  position: relative;
+}
+
+.kill-results {
+  position: absolute;
+  top: 0;
+  right: 0;
+  text-align: center;
+  vertical-align: middle;
+  z-index: 999;
+}
+
+.mapboxgl-popup-anchor-bottom {
+  padding-bottom: 20px;
+}
+
+.mapboxgl-popup-anchor-bottom .mapboxgl-popup-tip {
+  border-top-color: @brand-primary;
+}
+
+.mapboxgl-popup-content {
```

Comment:
Right now the popovers coverup the icons.  Can we add a bit margin on the bottom (~30px) to prevent this?